pyproject.toml

```diff
@@ -243,7 +243,9 @@ exclude = [
     "venv",
     "tests/assets",
 
-    "src/otx/core/engine/utils/*"   # It will be clean up later
+    # it will be cleaned up later
+    "src/otx/core/engine/utils/*",
+    "src/otx/algo/*",
```

Comment:
Could you explain why we need to exclude algo folder

Reply:
There are so many things to change for the module for now and most of the modules come from the OTX1.x. 
Since we didn't set the final rule of the pre-commit, I want to resolve this pre-commit at the later stage. @harimkang

Reply:
If so, I'd suggest adding this to the backbone folder only, just to be clear.

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/2678/commits/67c4a33cd0cbaa3858f62a0dde8e845d5da1704c